slz_testHarness.js

```diff
@@ -309,29 +305,21 @@ slz_TestLogger.createStandardProps = function () {
     this.allLogs = [];
 }
 
-slz_TestLogger.log = function (message) {
```

Comment:
Test Logger should be left out of this commit, as the Test Logger has been rewritten and decoupled

Reply:
This branch as been merged into feature/rebuild
Please pull changes from feature/rebuild into your branch and resolve locally, resubmit pr to feature/rebuild